week3/src/components/Content.jsx

```diff
@@ -0,0 +1,219 @@
+import React, { useEffect, useState } from 'react'
+import styled, {css, keyframes} from 'styled-components'
+
+const Section = styled.section`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: column;
+`;
+
+const Score = styled.header`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 100px;
+    height: 50px;
+
+    ${props => props.score == 1 &&
+        css`
+            background-color: #f29087;
+            animation-name: pop;
+            animation-duration: .2s;
+
+            @keyframes pop {
+                0% {
+                transform: translate(0, 0);
+                }
+                50% {
+                transform: translate(0, -30px);
+                }
+                100% {
+                transform: translate(0, 0);
+                }
+            }
+        `
+    }
```

Comment:
score 를 인자로 받고, css`` 문을 리턴하는 함수를 구성해서 분리할 수 있지 않을까 하네요!

```suggestion
    ${props => getAnimationCssByScore(props.score)}
```

이런 식으로 1~5 분기처리를 선언적으로 구현할 수 있을 것 같아요
되려나!!! 꼭 해봐욥

Reply:
아하 이렇게 함수로 처리해줄 수 있구나!! 고맙습니다🙆🙆